templates/admin/lbf/edit_lbf/partials/modify_group.html.twig

```diff
@@ -0,0 +1,37 @@
+<!-- template DIV that appears when user chooses to add a new group -->
+<div class="modal" id="{{ id }}group" tabindex="-1" aria-labelledby="{{ id }}GroupModal" aria-hidden="false">
+    <div class="modal-dialog modal-lg modal-dialog-scrollable">
+        <div class="modal-content">
+            <div class="modal-header">
+                {% set modalTitle = (id == "rename") ? "Rename Group" : "Add New Group" %}
+                {% set btnSaveValue = (id == "rename") ? "Rename Group" : "Save New Group" %}
```

Comment:
note i do not know the solution, but the 2 lines of code above will be impossible to capture in the translation pipeline (those 3 constants). Prob not a big deal hear since i bet these 3 constants are already in the translation list. But generally always best to partner a constant directly with an xl function so it gets captured in the translation pipeline (ie. a perl script that seeks them out; btw, reminds me that we will need to update that script to find the new formatted translation constants in twig templates; shouldn't be tough).

Reply:
Noted, I'll look into a solution